Modules/Video/BridgeOpenCV/include/itkOpenCVBasicTypeBridge.h

```diff
@@ -167,11 +167,11 @@ struct OpenCVBasicTypeBridge<itk::Size<2>, cv::Size>
   }
 };
 
-template <typename T, unsigned int NRows, unsigned int NColumns>
-struct OpenCVBasicTypeBridge<itk::Matrix<T, NRows, NColumns>, cv::Matx<T, NRows, NColumns>>
+template <typename T, unsigned int VRows, unsigned int VColumns>
```

Comment:
In this case, the old `NRows` and `NColumns` already suggested that these are the _number of_ rows and columns, respectively. When using the `V` initial instead, we might need to make it a little bit clearer, by naming them `VNumberOfRows`, `VNumberOfColumns`. What do you think?

Reply:
I thought that the old `NRows`meant "non-type rows" or "numeric rows" rather than "number of rows".  I am thinking to leave `VRows` as is.

Reply:
OK!